pom.xml

```diff
@@ -55,6 +55,11 @@
       <artifactId>workflow-step-api</artifactId>
       <version>1.15</version>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20090211</version>
```

Comment:
That's my bad. I meant to update this before I created the PR. I believe it was just the most popular version, which is why I started with this. I'll update this to a newer version